app/src/main/java/edu/uc/coffeens/stocktracker/dao/IStockDAO.kt

```diff
@@ -12,8 +12,4 @@ interface IStockDAO {
 
     @GET("/stevekinney/f96d5800852e91282f46/raw/ea047c2f5898de6c9fecf535db8b30abcfe5b423/stocks.json")
     fun getAllStocks(): Call<ArrayList<Stock>>
-
-    @GET("/stevekinney/f96d5800852e91282f46/raw/ea047c2f5898de6c9fecf535db8b30abcfe5b423/stocks.json")
-    fun getStock(@Query("Combined_Name") name: String): Call<ArrayList<Stock>>
-
```

Comment:
this function was never called/used at the time of review. 
I went ahead and removed it as it was not needed.

Reply:
Good.  Unused code is clutter.